regulations/urls.py

```diff
@@ -14,7 +15,11 @@
     path('', HomepageView.as_view(), name='homepage'),
     path('<numeric:part>/', RegulationLandingView.as_view(), name="regulation_landing_view"),
     path('<numeric:part>/<version:version>/', PartReaderView.as_view(), name='part_reader_view'),
-    path('<numeric:part>/<numeric:section>/<version:version>/', SectionReaderView.as_view(), name='section_reader_view'),
     path('<numeric:part>/<subpart:subpart>/<version:version>/', SubpartReaderView.as_view(), name="subpart_reader_view"),
+    path('<numeric:part>/<numeric:section>/<version:version>/', SectionReaderView.as_view(), name='section_reader_view'),
```

Comment:
Should be able to remove these. `section_reader_view` is referenced within:

- content/regulations_landing_pages/landing_433.html
- regulations-site/regulations/views/goto.py
- regulations-site/regulations/views/reader.py
- regulations-site/regulations/tests/urls_test.py
- regulations-site/regulations/templates/regulations/recent-changes.html
- regulations-site/regulations/templatetags/layers.py
- regulations-site/regulations/generator/section_url.py